pom.xml

```diff
@@ -82,7 +82,12 @@ under the License.
       <dependency>
         <groupId>org.apache.maven.shared</groupId>
         <artifactId>maven-verifier</artifactId>
-        <version>1.8.0</version>
+        <version>2.0.0-SNAPSHOT</version>
```

Comment:
I prefer a some of release version, even milestone
In other case tests can be unstable and investigation can be more difficult.